packages/api/src/providers/provider.ts

```diff
@@ -17,6 +17,8 @@ export enum ConsumerHealthDataType {
 export type DAPIParams = {
   timezoneId?: string;
   accessToken?: string;
+  xTenoviApiKey?: string;
+  xTenoviClientName?: string;
```

Comment:
Let's not add provider-specific to this generic code, please. We should be able to have the Tenovi code to receive a specialization of `DAPIParams`.